card/mod/admin/set/self/version.rb

```diff
@@ -2,8 +2,8 @@
 
 require_dependency "card/version"
 
-view :raw do |_args|
+def content
   Card::Version.release
 end
 
-view :core, :raw
+# view :core, :raw
```

Comment:
These aliases break without a raw view defined in this module.

I want to suggest that if a "hard alias" is impossible, we should consider a "soft" alias (which just generates something like:
```
  view :core do
    render_raw! :raw
  end
```
The use in this case (and several other cases) is just that it avoids having to run "process_content", because we know there's nothing to parse.  Might be better ways...

Reply:
eg `chunk_types nil` at the top of the module...